### PR TITLE
docs: Update link references of ownership

### PR DESCRIPTION
**Pull request by chinyeungli**
Update link references of ownership from `nexB` to `aboutcode-org`

### CODE REVIEW

**Comment by chinyeungli** [comment]
I apologize for the subpar message. I have revised the commit message. Kindly review it to ensure its appropriateness.

**Comment by sschuberth** [comment]
Sorry, yet more style issues: [Body text lines need to hard-wrapped at column 74](https://github.com/oss-review-toolkit/ort/actions/runs/10630948109/job/29547887090?pr=9053#step:4:19).

**Comment by sschuberth** [comment]
> nexB has transferred the public repositories to aboutcode-org

But *which* public repositories? I can still see other public repositories at https://github.com/nexB/, so apparently not all were moved. So what does "*the* public repositories" refer to? Which ones? What is the logic after which some public repositories are moved, and others not. That's what you should explain in the commit message.

**Comment by chinyeungli** [comment]
@sschuberth All the public repositories under nexB has been moved. Sorry, I should have created the PR after completed the migration process.

**Comment by sschuberth** [comment]
> Sorry, I should have created the PR after completed the migration process.

No worries, just making it explicit in the commit message that this is the plan would have suficed 😉

File: plugins/scanners/scancode/src/test/assets/scancode-3.0.2_mime-types-2.1.18.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "--strip-root": true,
         "--timeout": "300.0"
       },
-      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from aboutcode-org Inc. and others.\nVisit https://github.com/aboutcode-org/scancode-toolkit/ for support and download.",
```
Comment (sschuberth) [inline plugins/scanners/scancode/src/test/assets/scancode-3.0.2_mime-types-2.1.18.json:20]: I guess "nexB Inc." should stay as-is here, as "aboutcode-org" is not an "Inc.", or?

Comment (sschuberth) [reply plugins/scanners/scancode/src/test/assets/scancode-3.0.2_mime-types-2.1.18.json:20]: And actually, at the time this file was created, this was the correct output from ScanCode, so I believe the file should stay as-is.

Similar for all other JSON result files below. IMO these should not be touched at all.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: plugins/scanners/scancode/src/test/assets/scancode-3.0.2_mime-types-2.1.18.json (hunk 1 above)
Comment (chinyeungli) [reply plugins/scanners/scancode/src/test/assets/scancode-3.0.2_mime-types-2.1.18.json]: You're absolutely correct. Revert the reference links for all the test json files.

Comment (sschuberth) [reply plugins/scanners/scancode/src/test/assets/scancode-3.0.2_mime-types-2.1.18.json]: Thanks. Please squash your second commit into the first one, as our final Git history should not contain commits of fixes that were discovered during PR review.

**Comment by chinyeungli** [comment]
Thanks.

**Pull request merged**